aten/src/ATen/core/op_registration/op_registration_test.cpp

```diff
@@ -282,13 +282,26 @@ TEST(OperatorRegistrationTest, whenRegisteringMultipleKernelsInSameOpCallAndCall
   }, "Could not run '_test::dummy' with arguments from the 'XLA'"
   " backend.");
 
+  expectThrows<c10::Error>([&] {
+    callOp(*op, dummyTensor(c10::DispatchKey::MLC));
+  }, "Could not run '_test::dummy' with arguments from the 'MLC'"
+  " backend.");
```

Comment:
FWIW, these tests are unnecessary, there isn't anything substantively different about the MLC case here